instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/history/MapboxHistoryTestRule.kt

```diff
@@ -32,23 +37,29 @@ import java.util.concurrent.CountDownLatch
  */
 class MapboxHistoryTestRule : TestWatcher() {
 
+    private val mapboxTestDirectoryName = "mapbox_test"
     private val directory: File =
         File(
             Environment.getExternalStoragePublicDirectory(
                 Environment.DIRECTORY_DOWNLOADS
             ),
-            "mapbox_test"
+            mapboxTestDirectoryName
         )
 
     lateinit var historyRecorder: MapboxHistoryRecorder
+    lateinit var context: Context
```

Comment:
can the rule retrieve app's context `InstrumentationRegistry.getInstrumentation().getTargetContext()` itself instead requiring to provide an activity?

Reply:
Yes, it can, thanks.